<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(125, 125, 125);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="hourLCD">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255,255,255);    
}</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteLCD">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255,255,255);    
}</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secondLCD">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255,255,255);    
}</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="imageDownloadButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Image Download/Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string>image Label</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteTimeZoneLCD">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255, 255, 255);    

}</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secondTimeZoneLCD">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255, 255, 255);    

}</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="hourTimeZoneLCD">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
    color: rgb(255, 255, 255);    

}</string>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipcodeEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>98119</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Weather Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>weather label</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherImageLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>110</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>weather Image label</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeMessageLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeZoneLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Current Time in Nairobi, Kenya</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageSlideShowLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
